src/scores/continuous/interval_impl.py

```diff
@@ -77,7 +77,7 @@ def quantile_interval_score(  # pylint: disable=R0914
     References:
         Gneiting, T., & Raftery, A. E. (2007). Strictly proper scoring rules, prediction,
         and estimation. Journal of the American Statistical Association, 102(477), 359-378.
-        Corollary 5.2. https://doi.org/10.1198/016214506000001437
+        Section 6.2. https://doi.org/10.1198/016214506000001437
```

Comment:
Update this to be 
Winkler, R. L. (1972). A Decision-Theoretic Approach to Interval Estimation. Journal of the American Statistical Association, 67(337), 187. https://doi.org/10.2307/2284720

Reply:
done